src/_scss/pages/dashboard/filters/_fiscalYears.scss

```diff
@@ -0,0 +1,29 @@
+ul.fiscal-years {
+    @include unstyled-list;
+    font-weight: $font-normal;
+    padding-top: rem(15);
+    .fiscal-years__wrapper {
+        @include display(flex);
+    }
+    .fiscal-years__left {
+        padding-right: rem(4);
+    }
+    .fiscal-years__right {
+        padding-left: rem(4);
```

Comment:
Is there a reason we can't just put padding on the right for each column and not have to specify which side something is on? Seems simpler for future when we have more than 2 columns (I assume we're increasing column count? Maybe we won't, it just seems like a lot of empty space on the right)